src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/MassTransitClientExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using MassTransit;
+using MassTransit.Monitoring;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ;
+
+/// <summary>
+/// MassTransitClientExtensions provides extension methods for configuring MassTransit in a client application.
+/// </summary>
+public static class MassTransitClientExtensions
+{
+    /// <summary>
+    /// Configures MassTransit with RabbitMQ integration for the client side,
+    /// using the same configuration used by the hosting environment.
+    /// </summary>
+    /// <param name="services">The client service collection.</param>
+    /// <param name="name">A unique name for the RabbitMQ instance.</param>
+    /// <param name="telemetry">Enables telemetry, which could be exported to either OpenTelemetry or Application Insights.</param>
+    public static void AddMassTransitClient(this IServiceCollection services, string name, bool? telemetry = false)
+    {
+        if (string.IsNullOrWhiteSpace(name))
+            throw new ArgumentException("The name must be a valid, non-empty string.", nameof(name));
+
+        var configuration = services.BuildServiceProvider().GetRequiredService<IConfiguration>();
+        var configurationSection = configuration.GetSection($"MassTransit:{name}");
```

Comment:
I believe the section (according to the other Aspire clients) should be `Aspire:MassTransit:Client`

Reply:
I've done something like this in my PR as well, which also allows to bind a section with the `name`.
https://github.com/CommunityToolkit/Aspire/pull/277/files#diff-bfd900a6101f7a145a030e21810fac284c3397f522f7adeb3ade6c2bb976ad6bR59